<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Overview</class>
 <widget class="QWidget" name="Overview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Express Delivery</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
   font-size:20px;
}

QGroupBox{
   border:none;
}

QTextBrowser
 {
    font-size: 30px;
    background-color: transparent;
    color: #2c3e50;
    border: none;
    border-radius: 8px;
    cursor: pointer;
    transition: background-color 0.3s;
}
QTextBrowser:hover {
    background-color: #2980b9;
}

QTableView {
    background-color: transparent;
    color: #2c3e50;
    border: 1px solid #2c3e50; 
    border-radius: 8px;
}
</string>
  </property>
  <widget class="QGroupBox" name="EmployeeBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="EmployeeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Employ√©s:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="EmployeNumber">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="EmployeeTable">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>821</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="EmployeeBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>1280</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTableView" name="DeliveryTable">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>821</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DeliveryLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Commandes:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="DeliveryNumber">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="EmployeeBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>1280</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="ProductLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>System</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Produits en Stock:</string>
    </property>
   </widget>
   <widget class="QTableView" name="ProductTable">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>821</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ProductNumber">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
